src/Pmi/Evaluation/Evaluation.php

```diff
@@ -272,6 +329,13 @@ public function getForm(FormFactory $formFactory)
             $options['constraints'] = $constraints;
             $options['attr'] = $attributes;
 
+            if ($type === 'radio') {
+                $options['expanded'] = true;
+                if (empty($this->data->{$field->name})) {
+                    $options['data'] = 'in-person';
+                }
```

Comment:
We currently don't have any other radio button fields, so this works, but can we find a way to make this work without specifying this specific value as the default here?

A few options that come to mind:
1. Have the default option value be the empty value. At least in our case here, we really only care about whether it's EHR or not. So a default empty value might work and be a more generalization solution. You might need to play around with the [`empty_data`](https://symfony.com/doc/current/reference/forms/types/choice.html#empty-data) option to get this to work.
2. Or, if it's better to have the in-person value be the actual option value, then maybe you can add an extra field in the JSON configuration to specify what the `empty_data` value is.